go.mod

```diff
@@ -38,6 +38,7 @@ require (
 	github.com/go-openapi/swag v0.22.3 // indirect
 	github.com/gogo/protobuf v1.3.2 // indirect
 	github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da // indirect
+	github.com/golang/protobuf v1.5.3 // indirect
```

Comment:
This is still used here.

Reply:
Understood, it is still pulled in as an indirect dependency by a number of other components.

Reply:
@xing-yang Any chance this can be approved? I removed the explicit dependency on  github.com/golang/protobuf; the remaining indirect reference comes from other components that still use github.com/golang/protobuf.
If I am missing other steps that I should do as part of this PR to remove the indirect dependency, please let me know. Thanks.

Reply:
Can you check who is making this indirect dep?  You can try this "go mod why -m <lib>".

Reply:
It looks like it is being pulled in by github.com/container-storage-interface/spec/lib/go/csi

Reply:
Thanks for checking!